SeaPublicWebsite/ExternalServices/BreApi.cs

```diff
@@ -6,16 +6,19 @@
 using System.Net.Http.Headers;
 using System.Security.Cryptography;
 using System.Text;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
 using Newtonsoft.Json.Linq;
 using SeaPublicWebsite.DataModels;
+using SeaPublicWebsite.ExternalServices.Models;
 using SeaPublicWebsite.Helpers;
 using SeaPublicWebsite.Services;
 
 namespace SeaPublicWebsite.ExternalServices
 {
     public static class BreApi
     {
-        public static List<Recommendation> GetRecommendationsForUserRequest(string requestString)
+        public static async Task<List<Recommendation>> GetRecommendationsForUserRequest(BreRequest request)
```

Comment:
Please add `Async` to the end of this function name (it's a good idea for pretty much any async function so people using it know to `await` it)